stock_request_purchase/tests/test_stock_request_purchase.py

```diff
@@ -233,6 +233,8 @@ def test_view_actions(self):
 
     def test_create_request_analytic_data(self):
         """Single Stock request with analytic data"""
+        if 'analytic_account_id' not in self.env["stock.request"]:
```

Comment:
This shouldn't be necessary, but just in case:
```suggestion
        if 'analytic_account_id' not in self.env["stock.request"] or `analytic_tag_ids` not in self.env["stock.request"]:
```